src/test/java/org/apache/accumulo/proxy/its/SimpleProxyBase.java

```diff
@@ -1738,17 +1639,15 @@ public void userPermissions() throws Exception {
       assertFalse(client.listTables(creds).contains("fail"));
     }
     // denied!
-    try {
-      if (isKerberosEnabled()) {
-        // Switch back to the extra user
-        UserGroupInformation.loginUserFromKeytab(otherClient.getPrincipal(),
-            otherClient.getKeytab().getAbsolutePath());
-        client = userClient;
-      }
-      String scanner = client.createScanner(user, tableName, null);
-      client.nextK(scanner, 100);
-      fail("stooge should not read table test");
-    } catch (AccumuloSecurityException ex) {}
+    if (isKerberosEnabled()) {
+      // Switch back to the extra user
+      UserGroupInformation.loginUserFromKeytab(otherClient.getPrincipal(),
+          otherClient.getKeytab().getAbsolutePath());
+      client = userClient;
+    }
+    String scanner = client.createScanner(user, tableName, null);
+    assertThrows(AccumuloSecurityException.class, () -> client.nextK(scanner, 100),
+        "stooge should not read table test");
```

Comment:
The block was removed, causing variable name collisions for some variables, like `scanner`. If you prefer to reuse the variable names instead of renaming them, you can put them inside an anonymous block (keep the block, but drop the `try` and `catch`), as in:

```java
    {
      String scanner = client.createScanner(user, tableName, null);
      assertThrows(AccumuloSecurityException.class, () -> client.nextK(scanner, 100),
          "stooge should not read table test");
    }
```

Or you can just use a different variable name, like you did. It's up to you. I just wanted to make sure you were aware. I sometimes use anonymous blocks to more strictly control the scope of commonly named temporary variables, so they don't get accidentally referenced later.